ghcide/test/exe/Main.hs

```diff
@@ -1150,6 +1150,33 @@ removeImportTests = testGroup "remove import actions"
             , "type T = K.Type"
             ]
       liftIO $ expectedContentAfterAction @=? contentAfterAction
+  , testSession "remove unused operators whose name ends with '.'" $ do
+      let contentA = T.unlines
+            [ "module ModuleA where"
```

Comment:
Optional suggestion: I just realized there's (.|.) operator in Data.Bits in base, so we could potentially shrink the reproducer to just one module if you fancy minimalism :wink: 

```haskell
{-# OPTIONS_GHC -Wunused-imports #-}
module ModuleA where
import Data.Bits ((.|.), xor)
x=xor
```